src/components/registration/steps.jsx

```diff
@@ -182,6 +182,9 @@ class UsernameStep extends React.Component {
                         this.props.description
                     ) : (
                         <intl.FormattedMessage id="registration.usernameStepDescription" />
+                        <b>
```

Comment:
I think it'll be fine if you just put a `<span>...</span>` around the whole message.

Reply:
Sure, I can try doing that!